openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -205,7 +205,8 @@ def celebrations(self):
         Returns a list of celebrations that should be performed.
         """
         return {
-            'first_section': CourseEnrollmentCelebration.should_celebrate_first_section(self.enrollment_object),
+            'first_section': (CourseEnrollmentCelebration.should_celebrate_first_section(self.enrollment_object)
+                              if courseware_mfe_first_celebration_is_active(self.course_key) else False),
```

Comment:
This could be written as `is_active and should_celebrate` rather than the if/else, which might be clearer? I dunno, matter of taste.

But actually... Why not put the waffle check into the utility method `should_celebrate_first_section`? And save the rest of the utility methods in this PR as class methods for CourseEnrollmentCelebration? That way we won't have two methods that look like the same thing -- "is first celebration active" and "should we do first celebration" are confusingly similar.